### PR TITLE
feat: home page layout

### PR DESCRIPTION
**Pull request by valeryiay**
1. Task: [Implementing a main page](https://github.com/users/vetalapo/projects/3/views/1?pane=issue&itemId=67059890)
2. Screenshot:
image
image

Updated the images, but the cards are not active links

3. Here's what to think about:
- [x] Routing: check out
- [ ] Breadcrumbs
- [ ] Buttons: adding items to their cart, signing up, go to the Detailed Page
- [ ] Show Off Popular Stuff: Put the cool and popular stuff front and center. Show pictures and short descriptions to catch people's interest and make them want to see more.
- [ ] Search bar product.description?.["en-GB"]
- [ ] Product Categories from commercetoolsApi
- [ ] Special Offers (discont)
- [x] Responsive Design: layout is responsive and adapts seamlessly to various screen sizes and devices. ++ optimize images
- [ ] fast loading, smooth
- [ ] customer reviews and ratings for products directly into the main page, highlight positive reviews
4. This is what our home page might look like.  Logic and routing not added, can be developed

### CODE REVIEW

File: src/views/HomeView.vue [hunk 1]
```diff
@@ -1,18 +1,323 @@
...
+                    </div>
+                </v-card>
+            </v-col>
+        </v-row>
     </v-container>
 </template>
 
 <style scoped>
-    .main-container {
+    .catalog-container {
```
Comment (vetalapo) [inline src/views/HomeView.vue:155]: This container is no a catalog. Please think of a better name reflecting the container's purpose.

File: src/views/HomeView.vue [hunk 2]
```diff
@@ -1,18 +1,323 @@
...
+            toggleHover(index: number) {
+                this.categories[index].active = !this.categories[index].active;
+            }
+        }
+    });
 </script>
 
 <template>
-    <v-container class="main-container" fluid>
-        <TheWelcome />
```
Comment (vetalapo) [inline src/views/HomeView.vue:7]: We probably do not need "TheWelcome" component at all, you could delete it also.

**Review by vetalapo — CHANGES_REQUESTED** [review]
It's great to have something on the home page. Could you please also make categories pictures to be the same size?
It's looking crooked at the moment.

image

**Comment by JoMarianek** [comment]
Updated the images to look more uniform, let me know if you prefer all images to be vertical.

**Review by vetalapo — CHANGES_REQUESTED** [review]
I would suggest cards to be like this:
image

Without spaces in card because of horizontal pictures.
Spaces don't look attractive

image

**Review by vetalapo — CHANGES_REQUESTED** [review]
TheWelcome component, its test and entire components folder have to be removed as it's not used by anything else anymore.

**Review by vetalapo — APPROVED** [review]
Job nicely done!

**Pull request merged**